packages/nuxt3/src/components/loader.ts

```diff
@@ -24,13 +25,14 @@ export const loaderPlugin = createUnplugin((options: LoaderOptions) => ({
   }
 }))
 
-function findComponent (components: Component[], name:string) {
-  return components.find(({ pascalName, kebabName }) => [pascalName, kebabName].includes(name))
+function findComponent (components: Component[], name: string) {
+  const id = pascalCase(name)
```

Comment:
Not sure what was main issue but we are normalizing " and ' in names within component scanner for `pascalName`: 

```suggestion
  const id = pascalCase(name).replace(/["']/g, '')
```

Reply:
I'm not sure it's possible for a name to have these characters in at this point in the lifecycle, but it doesn't hurt.